scripts/create_sample_worksheet.py

```diff
@@ -338,9 +523,12 @@ def main():
         cleanup(cl, SampleWorksheet.TAG)
         return
     print(args)
-    ws = SampleWorksheet(cl, args)
+
+    ws = SampleWorksheet(cl, args.large, args.preview)
     start_time = time.time()
     ws.create()
+    if args.test:
+        ws.validate_content()
```

Comment:
Use the same name instead of two (`test` versus `validate`).

Reply:
I renamed it to `test_print`.